overlord/devicestate/firstboot.go

```diff
@@ -230,6 +230,7 @@ func populateStateFromSeedImpl(st *state.State, opts *populateStateFromSeedOptio
 		configTss = chainTs(configTss, snapstate.ConfigureSnap(st, "core", snapstate.UseConfigDefaults))
 	}
 
+	dangerousModel := model.Grade() == asserts.ModelDangerous
```

Comment:
Nitpick: I'd switch the words, otherwise it implies it's an model, and not a boolean
```suggestion
	modelDangerous := model.Grade() == asserts.ModelDangerous
```

Reply:
or `modelIsDangerous`